### ISSUE TITLE
Не обновляется информация после копирования

### ISSUE THREAD

**Issue by proninyaroslav** [whoozle/android-file-transfer-linux#60]
Здравствуйте. Заметил такую вещь: если копировать файл перетаскиванием на устройство, то в списке программы он не отображается, но физически копируется.

**Comment by whoozle** [comment]
А если refresh (f5) нажать потом?

**Comment by whoozle** [comment]
скажите что у вас за устройство и если есть что-то подозрительное в логах, их тоже сюда.

**Comment by proninyaroslav** [comment]
Galaxy S4 i9505. В логах пока ничего не заметил.

**Comment by proninyaroslav** [comment]
Оказалось что сам архив был битый. Но обновление списка не помешало бы всё равно.

**Comment by whoozle** [comment]
проверьте пожалуйста, обновляет ли если после загрузки выбрать в меню Refresh или нажать F5? Надо понять, это UI не обновляется или MTP server разорвало. :)

**Comment by proninyaroslav** [comment]
Нет, если перезапустить (или переключится с внутренней на внешнюю карту в списке), то обновляется.

**Comment by whoozle** [comment]
не могли бы вы протестировать на последнем мастере? Очень много воды утекло
